fakenet/listeners/FTPListener.py

```diff
@@ -190,15 +206,33 @@ def chdir(self, path):
         if not self.lexists(path):
             path = '.'
 
+        # Collect NBIs
+        nbi = {"command": "CWD", "path": path}
+        collect_nbi(self.cmd_channel.remote_port, nbi,
+                self.cmd_channel.server.config.get('usessl'),
+                self.cmd_channel.server.diverterListenerCallbacks)
+
         return super(FakeFS, self).chdir(path)
 
     def remove(self, path):
 
+        # Collect NBIs
+        nbi = {"command": "DELETE", "filename": path}
```

Comment:
We are interested in the virtual file path used by AbstractedFS class. In order to get the actual path seen by FakeNet, we need something like `AbstractedFS.fs2ftp(self, path)` (need to remove the forward slash as well from the filename returned)